alerts/etcMemberDown.md

```diff
@@ -42,13 +42,31 @@ the master nodes. This is the case when the [machine-config-operator
 $ oc get nodes -l node-role.kubernetes.io/master= -o template --template='{{range .items}}{{"===> node:> "}}{{.metadata.name}}{{"\n"}}{{range $k, $v := .metadata.annotations}}{{println $k ":" $v}}{{end}}{{"\n"}}{{end}}'
 ```
 
-## Mitigation
+### General etcd health
 
-If an upgrade is in progress, the alert may automatically resolve in some time
-when the master node comes up again. If MCO is not working on the master node,
-check the AWS to verify if the master node instances are running or not. AWS
-instance retirement might need a manual reboot of the master node.
+To run `etcdctl` commands, we need to `rsh` into the `etcdctl` container of any
+etcd pod.
+
+```console
+$ oc rsh -c etcdctl -n openshift-etcd $(oc get po -l app=etcd -oname -n openshift-etcd | awk -F"/" 'NR==1{ print $2 }')
```

Comment:
Markdown linter is failing here, I don't think we should break the command in two just to keep with line number size.